cmd/console.go

```diff
@@ -22,28 +22,33 @@ THE SOFTWARE.
 package cmd
 
 import (
-	"github.com/on4kjm/morserino_display/pkg/morserino_core"
+	"fmt"
+	"os"
+
+	"github.com/on4kjm/morserino_display/pkg/morserino"
 	"github.com/spf13/cobra"
 )
 
 // consoleCmd represents the console command
 var consoleCmd = &cobra.Command{
 	Use:   "console",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "Displays the Morserino output to the console",
 	Run: func(cmd *cobra.Command, args []string) {
-		morserino_core.Morserino_console(morserinoPortName)
+		err := morserino.SetupLogger(morserinoDebugLevel, morserinoDebugFilename)
+		if err != nil {
+			fmt.Println("\nERROR:  " + err.Error() + "\n")
+			cmd.Help()
+			os.Exit(1)
+		}
```

Comment:
Style nit: you can inline this.
Also `Println`: 
- already adds a `/n` at the end of the line. 
- Joins all given arguments with a whitespace character.
```suggestion
		if err := morserino.SetupLogger(morserinoDebugLevel, morserinoDebugFilename); err != nil {
			fmt.Println("\nERROR:",  err)
			cmd.Help()
			os.Exit(1)
		}
```